packages/react-core/src/components/FormSelect/examples/FormSelect.md

```diff
@@ -258,3 +258,41 @@ class FormSelectInputGrouped extends React.Component {
   }
 }
 ```
+
+### Icon sprite variant
+
+**Note:** The dropdown toggle icon is applied as a background image to the form element. By default, the image URLs for these icons are data URIs. However, there may be cases where data URIs are not ideal, such as in an application with a content security policy that disallows data URIs for security reasons. The `.pf-m-icon-sprite` variation changes the icon source to an external SVG file that serves as a sprite for all of the supported icons.
+
+```js isBeta
+
+import React from 'react';
+import { FormSelect, FormSelectOption, FormSelectOptionGroup } from '@patternfly/react-core';
+
+class FormSelectIconSpriteInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: ''
+    };
+    this.onChange = (value, event) => {
+      this.setState({ value });
+    };
+    this.options = [
+      { value: '', label: 'Select a number', disabled: false, isPlaceholder: true  },
+      { value: '1', label: 'One', disabled: false, isPlaceholder: false  },
+      { value: '2', label: 'Two', disabled: false, isPlaceholder: false  },
+      { value: '3', label: 'Three - the only valid option', disabled: false, isPlaceholder: false  }
+    ];
+  }
+
+  render() {
+    return (
+      <FormSelect isIconSprite value={this.state.value} onChange={this.onChange} aria-label="FormSelect Input">
```

Comment:
The contents in the select give the impression it's validated, but it doesn't seem to change state with different selections. It would be good to show the different validation states so we can see/verify the icon sprite in those states - like the other examples.